### ISSUE TITLE
CRS mismatch error when trying to map shorelines with manually-uploaded transects

### ISSUE THREAD

**Issue by FlorisCalkoen** [SatelliteShorelines/CoastSeg#247]
ping #openjournals/joss-reviews#6683

I have manually uploaded transects for my area of interest in 4326 - please see output below, but I get a CRS mismatch error, when computing the shoreline distances. Can you reproduce this? I'm looking at Terschelling in The Netherlands (see screenshot below). 

```ipython
In [5]: transects = gpd.read_file("https://coclico.blob.core.windows.net/public/gcts-terschelling.geojson")

In [6]: transects.crs
Out[6]:
<Geographic 2D CRS: EPSG:4326>
Name: WGS 84
Axis Info [ellipsoidal]:
- Lat[north]: Geodetic latitude (degree)
- Lon[east]: Geodetic longitude (degree)
Area of Use:
- name: World.
- bounds: (-180.0, -90.0, 180.0, 90.0)
Datum: World Geodetic System 1984 ensemble
- Ellipsoid: WGS 84
- Prime Meridian: Greenwich
```

```
CRS mismatch between CRS of the passed geometries and 'crs'. Use 'GeoDataFrame.set_crs(crs, allow_override=True)' to overwrite CRS or 'GeoDataFrame.to_crs(crs)' to reproject geometries.

Additional Information
Traceback (most recent call last): File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/map_UI.py", line 905, in extract_shorelines_button_clicked self.coastseg_map.extract_all_shorelines() File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/coastseg_map.py", line 1858, in extract_all_shorelines self.compute_transects(self.transects.gdf, self.get_settings(), selected_roi_ids) File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/coastseg_map.py", line 1992, in compute_transects self.save_transect_timeseries(session_path,self.rois.get_extracted_shoreline(roi_id),roi_id) File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/coastseg_map.py", line 2098, in save_transect_timeseries common.save_transects( File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/common.py", line 1912, in save_transects merged_timeseries_df,timeseries_df = add_lat_lon_to_timeseries(merged_timeseries_df, transects_gdf.to_crs('epsg:4326'),cross_distance_df, File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/common.py", line 1787, in add_lat_lon_to_timeseries new_gdf_shorelines_wgs84=convert_points_to_linestrings(cross_shore_pts, group_col='date', output_crs='epsg:4326') File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/coastseg/common.py", line 1998, in convert_points_to_linestrings linestrings_gdf = gpd.GeoDataFrame(linestrings, columns=['geometry'], crs=output_crs) File "/Users/calkoen/mambaforge/envs/coastseg/lib/python3.10/site-packages/geopandas/geodataframe.py", line 152, in __init__ raise ValueError(crs_mismatch_error) ValueError: CRS mismatch between CRS of the passed geometries and 'crs'. Use 'GeoDataFrame.set_crs(crs, allow_override=True)' to overwrite CRS or 'GeoDataFrame.to_crs(crs)' to reproject geometries.
```

![image](https://github.com/SatelliteShorelines/CoastSeg/assets/44444001/2873d24e-73ba-4179-830f-43880dba592f)

**Comment by 2320sharon** [comment]
Hi @FlorisCalkoen,

Sorry for the delay getting back to you. I was having some difficulty reproducing the error you showed here.

To attempt reproduce the error I did the following but I did not get the error you displayed here:
1. I loaded the sample transects you provided into a geodataframe and saved it to a geojson file

![image](https://github.com/SatelliteShorelines/CoastSeg/assets/61564689/3febc211-5c77-4331-b26c-3716dac735b9)

2. I used the load transects from file button to load in the transects

3. I drew a bounding box and loaded the default shoreline available in the region

4. I downloaded a small subset of imagery and extracted shorelines with no error (I did have to turn off cloud masking due to the cloud masking algorithm for LandSat masking out the shoreline)

5. I ran all these steps and did not get any error messages.

I ran all these steps on CoastSeg version 1.2.4 on a windows 11 computer.

Could you run `conda list coastseg` in your coastseg environment so I could see which coastseg version you're using?

I'm also going to attach the transect files I used.

[transects.zip](https://github.com/SatelliteShorelines/CoastSeg/files/15238665/transects.zip)

**Comment by 2320sharon** [comment]
Hi @FlorisCalkoen,

Sorry for the delay getting back to you. I was having some difficulty reproducing the error you showed here, until I realized I was running CoastSeg 1.2.2 instead of 1.2.4 . I'm currently working on understanding this error and fixing it. Thanks for finding this

**Comment by 2320sharon** [comment]
@FlorisCalkoen 

Good news! I was able to replicate your bug and its now fixed in CoastSeg 1.2.5 .  Once you've updated your coastseg environment this issue should be fixed

**Comment by FlorisCalkoen** [comment]
@2320sharon, many thanks for the fix, as I can confirm that with CoastSeg 1.2.5 I can map SDS with my own transects.

**Comment by 2320sharon** [comment]
@FlorisCalkoen that's great to hear! I'm going to close this issue now that it has been fixed.
